tests/test_basics.py

```diff
@@ -222,6 +222,29 @@ def test_breadcrumbs(sentry_init, capture_events):
     assert len(event["breadcrumbs"]["values"]) == 0
 
 
+def test_attachments(sentry_init, capture_envelopes):
```

Comment:
I think it would be nice to also test the attachments with an actual file on disk.

Reply:
Added.